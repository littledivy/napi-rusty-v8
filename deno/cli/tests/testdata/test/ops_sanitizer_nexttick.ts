import { nextTick } from "../../../../test_util/std/node/_next_tick.ts";

// https://github.com/denoland/deno_std/issues/1651

Deno.test("test 1", async () => {
  await new Promise<void>((resolve) => nextTick(resolve));
});

Deno.test("test 2", async () => {
  await new Promise<void>((resolve) => nextTick(resolve));
});
