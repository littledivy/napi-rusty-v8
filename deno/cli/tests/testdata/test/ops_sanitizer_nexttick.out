Check [WILDCARD]/testdata/test/ops_sanitizer_nexttick.ts
running 2 tests from [WILDCARD]/testdata/test/ops_sanitizer_nexttick.ts
test test 1 ... ok ([WILDCARD])
test test 2 ... ok ([WILDCARD])

test result: ok. 2 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out ([WILDCARD])

